assembl/views/api2/discussion.py

```diff
@@ -730,6 +730,11 @@ def get_visitors(request):
     fieldnames = ["time", "name", "email"]
     extra_columns_info = (None if 'no_extra_columns' in request.GET else
                           load_social_columns_info(discussion, "en"))
+    db = discussion.db
+    from assembl.models import TextField
+    configurable_fields = db.query(TextField).filter(TextField.discussion_id == discussion.id).filter(TextField.identifier == "CUSTOM").all()
```

Comment:
This is not correct, you're retrieving only TextField, and not SelectField. You need to use `db.query(AbstractConfigurableField)`  here. For the SelectField value, it's an array of graphene_id, you will need to get the id from graphene id, and get the label of the field associated with it. concatenate the items with comma (for now it's only one value in the array, but maybe we will have a multivalued selectfield in the future)